src/ucp/api/ucp.h

```diff
@@ -3490,7 +3490,7 @@ ucs_status_ptr_t ucp_stream_recv_data_nb(ucp_ep_h ep, size_t *length);
  * buffer.  In order to notify the application about completion of the receive
  * operation the UCP library will invoke the call-back @a cb when the received
  * message is in the receive buffer and ready for application access.  If the
- * receive operation cannot be stated the routine returns an error.
+ * receive operation cannot be started the routine returns an error.
```

Comment:
think "started, then the ..." reads better

Reply:
done, also fixed for ucp_tag_send_sync_nbx, ucp_stream_recv_data_nb